tests/jvm/src/test/resources/metacp.owners

```diff
@@ -6,6 +6,108 @@ _empty_.B# => _empty_.
 _empty_.B#`<init>`(). => _empty_.B#
 _empty_.B#a(). => _empty_.B#
 _root_. => <none>
+_root_.com. => _root_.
+_root_.com.javacp. => _root_.com.
+_root_.com.javacp.ClassSuffix# => _root_.com.javacp.
+_root_.com.javacp.ClassSuffix#<init>(). => <none>
```

Comment:
Why? (Ditto for a bunch of other symbols below that also don't seem to have an owner).